library/validation/validate_core.php

```diff
@@ -71,6 +71,8 @@ function validateUsingPageRules($fileNamePath)
         print '<!---Start of page  form validation-->';
         print '<!--//include new rules of submitme functionallity-->';
         echo("\r\n");
+        //Not lbf forms use the new validation, please make sure you have the corresponding values in the list Page validation
+        $use_validate_js = 1;
         require_once($GLOBALS['srcdir'] . "/validation/validation_script.js.php");
         $new_validate = $GLOBALS['new_validate'] ? 1 : 0;
```

Comment:
@sharonco-matrix ,
I am confused by the line above setting $new_validate. Should this be removed and then use $use_validate_js on  line 103?